packages/firestore/src/core/sync_engine.ts

```diff
@@ -1008,11 +1008,12 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
     return activeQueries;
   }
   /**
-   * Creates a `Query` object from this `Target`. Note the result might be
-   * different from the original `Query` from which we obtained this instance.
+   * Creates a `Query` object from this `Target`. There is no way to obtain
+   * the original `Query`, so we synthesize a `Query` from the `Target` object.
+   * The result might be different from the original `Query`,
```

Comment:
Replace comma with period. Maybe also mention that is used for multi-tab and hence the limitation described here does not alter the behavior?